lib/vtls/gtls.c

```diff
@@ -833,8 +833,10 @@ Curl_gtls_verifyserver(struct Curl_easy *data,
                                      gnutls_cipher_get(session),
                                      gnutls_mac_get(session));
 
+#ifndef CURL_DISABLE_VERBOSE_STRINGS
   infof(data, "SSL connection using %s / %s",
         gnutls_protocol_get_name(version), ptr);
+#endif
```

Comment:
Should we also deal with the `ptr` value is assigned by unused with `--disabled-verbose`? Similar cases often cause a `-Wunused-variable`.

Reply:
Right, this function should not declare and assign `ptr` when not using it. @baruchsiach can you add that into this fix?

Reply:
> Right, this function should not declare and assign `ptr` when not using it. @baruchsiach can you add that into this fix?

Done.